firebase-perf/src/main/java/com/google/firebase/perf/network/InstrURLConnectionBase.java

```diff
@@ -38,11 +38,11 @@
 
   private static final String USER_AGENT_PROPERTY = "User-Agent";
 
-  private final HttpURLConnection mHttpUrlConnection;
-  private final NetworkRequestMetricBuilder mBuilder;
-  private long mTimeRequested = -1;
-  private long mTimeToResponseInitiated = -1;
-  private final Timer mTimer;
+  private final HttpURLConnection httpUrlConnection;
+  private final NetworkRequestMetricBuilder networkMetricBuilder;
+  private long timeRequested = -1;
+  private long timeToResponseInitiated = -1;
+  private final Timer timer;
```

Comment:
nit: We may want to group variables by modifiers and static/non-static variable for better readability. Here, we can group `private final` varibles together. No actions needed to check variables in other files because it is out of the scope of this PR, but this is something we can do when we examine the variables :)

Reply:
Done!